mosaicQueen/make_mosaic.py

```diff
@@ -227,7 +227,7 @@ def update_mos(mos, slc, image_regrid_hdu, beam_regrid_hdu, cutoff, noise, finit
 
 
 @profile
```

Comment:
Yesterday I tried to make it a command-line option but I don't really know how to do it since it is not an argument of any function and it does not live in `main`.

If you have no advice on how to do that sure, let's comment it out. We know where to find it when needed.